### PR TITLE
feat: migration - triple balance and claims

### PR DESCRIPTION
**Pull request by enthusiastmartin**

### CODE REVIEW

File: scripts/snakenet-migration/index.js [hunk 1]
```diff
@@ -205,6 +214,8 @@ const all_modules = [
     ["GenesisHistory.PreviousChain", "0x1754677a24055221d22db56f83f5e21390895d6c6b21a85c004b8942c3bc35ae"],
 ]
 
+const excludeFromTripling = ["7NPoMQbiA6trJKkjB35uk96MeJD4PGWkLQLH7k7hXEkZpiba"];
```
Comment (lumir-mrkva) [inline scripts/snakenet-migration/index.js:218]: what is this account?

GalactiCouncil `7HqdGVRB4MXz1osLR77mfWoo536cWasTYsuAbVuicHdiKQXf` and treasury should be excluded, we can add Alice, but it should be noted that is just for fork

File: scripts/snakenet-migration/index.js [hunk 2]
```diff
@@ -253,6 +264,7 @@ const downloadData = async (url, destination, block_number = undefined) => {
 
                     const values = JSON.parse(JSON.stringify(resp, null, 4));
 
+                    // filter alice
```
Comment (lumir-mrkva) [inline scripts/snakenet-migration/index.js:268]: if this is only related to testing fork, it should be noted

File: scripts/snakenet-migration/index.js [hunk 3]
```diff
@@ -325,6 +337,53 @@ const sendAndWaitFinalization = ({from, tx, printEvents = []}) => new Promise(re
...
+        const tripled = balance.imuln(3);
+        assert( bal.eq(tripled), `Incorrect amount for ${address}`);
+    }
+
+    let balances = [];
+
+    await api.query.system.account.entries().then( accounts => {
+        accounts.map( ([key, {data}]) => {
+            const [address] = key.toHuman()
+            //Skip Alice - only for testing purposes
```
Comment (lumir-mrkva) [inline scripts/snakenet-migration/index.js:358]: use the same **excluded** const

File: scripts/snakenet-migration/index.js [hunk 4]
```diff
@@ -358,16 +417,127 @@ const validate = async (source_url, target_url) => {
...
+        reserved: aInfo.data.reserved,
+        miscFrozen: aInfo.data.miscFrozen,
+        feeFrozen: aInfo.data.feeFrozen})
+
+    let newInfo = registry.createType("AccountInfo", { ...aInfo, data: newData});
+
+    return newInfo.toHex();
+}
+
+const tripleClaims = (registry, value) => {
```
Comment (lumir-mrkva) [inline scripts/snakenet-migration/index.js:528]: tripleClaims -> tripleClaim

**Review by lumir-mrkva — CHANGES_REQUESTED** [review]

File: scripts/snakenet-migration/index.js (hunk 1 above)
Comment (enthusiastmartin) [reply scripts/snakenet-migration/index.js:219]: No particular meaning. it was only for testing puporses, same as Alice address used. I will clean it up.

File: scripts/snakenet-migration/index.js (hunk 3 above)
Comment (enthusiastmartin) [reply scripts/snakenet-migration/index.js:359]: Yes, at this time , there was not yet the excluded list. Will get updated.

File: scripts/snakenet-migration/index.js [hunk 5]
```diff
@@ -358,16 +418,162 @@ const validate = async (source_url, target_url) => {
...
+    const updateLocks = locks.filter( (lock) => lock.id.toString() !== stakingId.toString());
+
+    return registry.createType("Vec<BalanceLock<Balance>>", updateLocks).toHex();
+}
+
+const tripleBalance = (registry, value) => {
+    let aInfo = registry.createType("AccountInfo", value);
+    let balance = new BN(aInfo.data.free.toString())
+
+    balance = balance.imuln(3);
```
[the remark this answers is not in the document]
Comment (enthusiastmartin) [reply scripts/snakenet-migration/index.js]: corrected

**Review by lumir-mrkva — APPROVED** [review]

**Comment by lumir-mrkva** [comment]
related to #360

**Pull request merged**